test/modules.coffee

```diff
@@ -0,0 +1,492 @@
+# Modules, a.k.a. ES2015 import/export
+# ------------------------------------
+#
+# Remember, we’re not *resolving* modules, just outputting valid ES2015 syntax.
+
+
+# This is the CoffeeScript import and export syntax, closely modeled after the ES2015 syntax
+# https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import
+# https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
+
+# import "module-name"
+# import defaultMember from "module-name"
+# import * as name from "module-name"
+# import { member } from "module-name"
+# import { member as alias } from "module-name"
+# import { member1 , member2 } from "module-name"
+# import { member1 , member2 as alias2 , [...] } from "module-name"
+# import defaultMember, { member [ , [...] ] } from "module-name"
+# import defaultMember, * as name from "module-name"
+
+# export { name1, name2, …, nameN }
+# export { variable1 as name1, variable2 as name2, …, nameN }
+# export name1 = …
+# export class …
+# export default expression
+# export default ->
+# export { name1 as default, … }
+
+# export * from …
+# export { name1, name2, …, nameN } from …
+# export { import1 as name1, import2 as name2, …, nameN } from …
+
+# Syntaxes from the MDN documentation that are *not* supported, because of ambiguous grammar and similarity to unsupported `var foo, bar = 'baz'`:
+# export name1, name2, …, nameN
+# export name1 = …, name2 = …, …, nameN
+
+
+# Helper function
+toJS = (str) ->
+  CoffeeScript.compile str, bare: yes
+  .replace /^\s+|\s+$/g, '' # Trim leading/trailing whitespace
+
+
+# Import statements
+
+test "backticked import statement", ->
+  input = "`import { foo, bar as baz } from 'lib'`"
+  output = "import { foo, bar as baz } from 'lib';"
+  eq toJS(input), output
```

Comment:
Legacy code that uses ES2015 modules probably has backticked `import` and `export` statements. I wanted to make sure we aren’t breaking such code. Perhaps it’s trivial to test specifically for this since we test backticks elsewhere. Do you think we should remove this test?